modules/core/src/parallel.cpp

```diff
@@ -942,7 +942,7 @@ int getNumberOfCPUs_()
 
 #endif
 
-#if !defined(_WIN32) && !defined(__APPLE__)
+#if !defined(_WIN32) && !defined(__APPLE__) && defined __GLIBC__
```

Comment:
m.b. `&& defined(_SC_NPROCESSORS_ONLN) ?

Reply:
right, that is much cleaner, thx !